.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+{
+    // Use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+    "build": {
+        "dockerfile": "Dockerfile"
+    },
+    "runArgs": [
```

Comment:
What are the defaults when this is commented out?

Reply:
Just the default like when you run a regular docker container.